xrpl/models/__init__.py

```diff
@@ -1,14 +1,22 @@
 """Top-level exports for the models package."""
 from xrpl.models import amounts, currencies, requests, transactions
+from xrpl.models.amounts import *  # noqa: F401, F403
+from xrpl.models.currencies import *  # noqa: F401, F403
```

Comment:
you don't need to do this. once you have `amounts` you can access `amounts.__all__`. as it is, this import doesn't do anything AFAICT

Reply:
That doesn't work (I just tried it to make sure) - it still needs to be available in the `__init__` file as an imported thing in order for it to be valid in its `__all__`.

Reply:
I tried in the console myself. Can you show what you tried? Maybe I'm missing something.

Reply:
I commented out one of the `import *` lines and then tried to import something from that directory from `xrpl.models`, and it had an `ImportError`

Reply:
oh i see why